website/static/js/comment.js

```diff
@@ -13,7 +13,57 @@ require('jquery-autosize');
 var osfHelpers = require('js/osfHelpers');
 var CommentPane = require('js/commentpane');
 var markdown = require('js/markdown');
+var atjsConfig = require('js/atjsConfig');
 
+var input = $('.atwho-input');
+var nodeId = window.nodeId;
+
+var getContributorList = function(input, nodeId) {
+    var url = osfHelpers.apiV2Url('nodes/' + nodeId + '/contributors/', {});
+    var request = osfHelpers.ajaxJSON(
+        'GET',
+        url,
+        {'isCors': true});
+    request.done(function(response) {
+        var activeContributors = response.data.filter(function(item) {
+            return item.embeds.users.data.attributes.active === true;
+        });
+        var data = activeContributors.map(function(item) {
+            return {
+                'id': item.id,
+                'name': item.embeds.users.data.attributes.given_name,
+                'fullName': item.embeds.users.data.attributes.full_name,
+                'link': item.embeds.users.data.links.html
+            };
+        });
+        // for any input areas that currently exist on page
+        input.atwho('load','@', data).atwho('load', '+', data).atwho('run');
+        // for future input areas so that data doesn't need to be reloaded
+        atjsConfig.at_config.data = data;
+        atjsConfig.plus_config.data = data;
+    });
+    request.fail(function(xhr, status, error) {
+        Raven.captureMessage('Error getting contributors', {
+            url: url,
+            status: status,
+            error: error
+        });
+    });
+    return request;
+};
+
+getContributorList(input, nodeId);
+input
+    .atwho(atjsConfig.at_config)
+    .atwho(atjsConfig.plus_config)
+    .bind('paste', atjsConfig.onPaste)
+    .on('focusin keyup', atjsConfig.lastElementBr)
+    .on('focusout', atjsConfig.onlyElementBr)
+    .keydown(function(e) {
+        if(e.which === 13 && !e.isDefaultPrevented()) {
+            atjsConfig.onReturn(e);
+        }
+    });
```

Comment:
I'll move it into the `init` function. It will only impact contenteditable fields with the `atwho-input` class though (`var input = $('.atwho-input');`).